lerobot/common/policies/diffusion/modeling_diffusion.py

```diff
@@ -749,3 +768,264 @@ def forward(self, x: Tensor, cond: Tensor) -> Tensor:
         out = self.conv2(out)
         out = out + self.residual_conv(x)
         return out
+
+
+class TransformerForDiffusion(nn.Module):
+    def __init__(self, config: DiffusionConfig, cond_dim: int):
+        super().__init__()
+        self.config = config
+
+        # compute number of tokens for main trunk and condition encoder
+        if config.n_obs_steps is None:
+            config.n_obs_steps = config.horizon
+
+        t = config.horizon
```

Comment:
Hmm, is it okay if we just leave this as config.horizon rather than binding it to another much less descriptive variable name?